package/standalone_file_logger/standalone_file_logger/src/rotating_logger.h

```diff
@@ -105,6 +114,15 @@ class RotatingLogger {
 
   FILE *_cur_file;
   size_t _bytes_written;
+
+
+  bool _finished = false;
```

Comment:
this should be a an atomic bool or flag.

Reply:
fixed